app/invocation/invocation_model.tsx

```diff
@@ -364,6 +364,13 @@ export default class InvocationModel {
     return this.optionsMap.get("remote_instance_name") ?? "";
   }
 
+  getDigestFunctionDir() {
```

Comment:
i will admit that i like this better without the slash, but i feel like i'm gonna cause a bug by asking for it.

Reply:
I also like it better without the slash, but am fine with keeping this as-is, since eventually I'd like to get rid of this func anyway and replace all of our manual URL construction with a cache util package

Reply:
same, but it makes concatenation logic at the callsites gnarlier and I assume whomever makes the url-lib will fix this one.